src/styles/Blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    justify-content: center;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    padding-left: 20px;
+  }
+
+  &_item:first-child {
+    padding-left: 0;
+  }
+
+  &__link {
+    position: relative;

```

Comment:
Try to organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)